ci/authn-azure/setup-conjur.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -ex
+
+sudo docker load -i conjur-image.tar
+
+# Remove old containers if they are up
+if [[ -d "conjur-quickstart" ]]
+then
+  pushd conjur-quickstart
+  sudo docker-compose down -v
+  popd
+fi
+
+rm -rf conjur-quickstart
+git clone --single-branch --branch local-conjur-image https://github.com/cyberark/conjur-quickstart.git
+cd conjur-quickstart
```

Comment:
For uniformity, let's choose "cd" or pushd"

Reply:
I `cd`'d because this is the only place that will take action. I use `pushd` when i do some actions in another place than the main area where the script is running. So I don't really need to `popd` after all the actions running in `conjur-quickstart`